scripts/pr_review_test.py

```
@@ -2,461 +2,331 @@
-        self.assertIn("Do not emit ten headings", pr_review.PROMPT_DEEP)
-        self.assertIn("fewer than 1,200 words", pr_review.PROMPT_DEEP)
-        self.assertIn("Never omit or merge independently material", pr_review.PROMPT_DEEP)
-        self.assertIn("at most three sentences", pr_review.PROMPT_DEEP)
-        self.assertIn("The `## Audit coverage` paragraph is still required", pr_review.PROMPT_DEEP)
-        self.assertNotIn("Answer all ten sections", pr_review.PROMPT_DEEP)
-        self.assertNotIn("even when a section has no finding", pr_review.PROMPT_DEEP)
-        self.assertIn(
-            "No material security or correctness issues found in the supplied static diff.",
-            pr_review.PROMPT_DEEP,
-        )
+            with self.assertRaises(pr_review.FetchError):
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)